robottelo/ui/locators/common.py

```diff
@@ -124,6 +124,10 @@
 
     "application_logo": (
         By.XPATH, "//img[contains(@alt, 'Header logo')]"),
+    "permission_denied": (
+        By.XPATH,
+        "//*[@id='content']/div[2]/h1[contains(.,'Permission denied')]"
```

Comment:
we do not use absolute paths as it is bad practice from all sides
Try to use:
`//h1[text()='Permission denied']`
or as last resort
`//h1[contains(.,'Permission denied')]`
but check that element is only ONE in Chrome debug tool or Firefox Firebug